includes/wikia/traits/PowerUserTrait.php

```diff
@@ -10,7 +10,7 @@
 
 trait PowerUserTrait {
 	// Required base-class methods
-	abstract function getBoolOption();
+	abstract function getBoolOption( $oname );
```

Comment:
`public function getBoolOption( $oname )` is how this method is defined in `User` class. Let's keep consistent :)